pkg/kor/kor.go

```diff
@@ -3,18 +3,18 @@ package kor
 import (
 	"bytes"
 	"fmt"
-	"os"
```

Comment:
The imports should be:
- standard library modules
- 3rd party modules
- current project modules

Reply:
I actually didn’t touch the imports. This must also be part of the automatic changes by my IDE.

I think golangci-lint had a linter for this exact problem.

Reply:
Just an update on this comment:
The [gci](https://golangci-lint.run/usage/linters/#gci) linter will take care of this now. Every time the linter runs it will check if the package import order is valid and fail the checks when it isn't.